### DISCUSSION TITLE
Thank dog for your clear cache options

### DISCUSSION THREAD

**Discussion by hb-webdev** [DylanVann/react-native-fast-image#848] [Show and tell]
I never had issues with wanting images to load quicker; I only display 1 to 3 at a time. I've been battling for months on Android with its extremely aggressive caching methods, never overwriting images with the same file name. (React Native iOS has no problems like this at all.) Your memory & disk clear options are the only thing that worked on one of my app screens. So thanks.

**Comment by fukemy** [comment]
who dog?
